lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -163,7 +163,7 @@ export abstract class ClientApplication {
             if (e instanceof AuthError) {
                 e.setCorrelationId(validRequest.correlationId);
             }
-            serverTelemetryManager.cacheFailedRequest(e);
+            serverTelemetryManager.cacheFailedRequest(e as AuthError);
```

Comment:
`serverTelemetryManager.cacheFailedRequest()` accepts any type of error. You don't need to cast it explicitly https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-browser-v3/lib/msal-common/src/telemetry/server/ServerTelemetryManager.ts#L82.